src/renderers/webgl/WebGLTextures.js

```diff
@@ -81,6 +81,16 @@ function WebGLTextures( _gl, extensions, state, properties, capabilities, utils,
 
 	}
 
+	function generateMipmap( texture, target ) {
+
+		_gl.generateMipmap( target );
+
+		var image = Array.isArray( texture.image ) ? texture.image[ 0 ] : texture.image;
```

Comment:
Can I expect images for cubic map have the same size?

Reply:
It's not guaranteed, but lets assume that.

Reply:
OK. I wanna add comment about our assumption here later.